eng/source-build-release-official.yml

```diff
@@ -113,12 +113,15 @@ stages:
   dependsOn: PreRelease
   jobs:
   - deployment: MirrorApproval
+    pool: server
     ${{ if or(eq(parameters.dotnetMajorVersion, '6.0'), eq(parameters.dotnetMajorVersion, '7.0')) }}:
       displayName: Confirm PR merged & ready for dotnet-security-partners mirroring
     ${{ else }}:
       displayName: Confirm ready for dotnet-security-partners mirroring
-    environment: Source Build Release - Mirror
-    pool: server
+    ${{ if parameters.isDryRun }}:
+      environment: pr
+    ${{ if not(parameters.isDryRun) }}:
```

Comment:
@mthalman I had that but it complained that `${{ else }}` is a key already defined for that object (it's few lines up). I think it might be just a YAML error but I wasn't going to risk that 🤷‍♂️